server/statistics/hot_peer_cache.go

```diff
@@ -308,8 +274,7 @@ func (f *hotPeerCache) getAllStoreIDs(region *core.RegionInfo) []uint64 {
 
 	// new stores
 	for _, peer := range region.GetPeers() {
-		// ReadFlow no need consider the followers.
-		if f.kind == ReadFlow && peer.GetStoreId() != region.GetLeader().GetStoreId() {
+		if region.GetLeader() != nil && region.GetLeader().StoreId != peer.StoreId && !includeFollowers {
```

Comment:
includeReadFollowers? Before this pr, only ReadFlow no need consider the followers